src/Symfony/Bridge/Doctrine/Test/DoctrineTestHelper.php

```diff
@@ -14,6 +14,7 @@
 use Doctrine\Common\Annotations\AnnotationReader;
 use Doctrine\ORM\Mapping\Driver\AnnotationDriver;
 use Doctrine\ORM\EntityManager;
+use PHPUnit\Framework\TestCase;
```

Comment:
ah indeed. What requires newer versions is the compat layer for more PHPUnit classes (which is only in 4.8.35 and 5.7). But I think this concerns only the bridge.